packages/iTwinUI-react/src/core/Wizard/Step.test.tsx

```diff
@@ -180,7 +181,7 @@ describe.each(['long', 'workflow'] as const)('Wizard step (%s)', (type) => {
 
     // Renders step
     const stepContainer = container.querySelector(
-      '.iui-wizard-step',
+      `.iui-${classNamePrefix}-step`,
```

Comment:
For `Wizard` components that are `type: default | long`, we are using `iui-stepper-*`. For `type: workflow`, we use `workflow-diagram`. Instead of making a conditional statement for each time I use a class name, I just created a variable at the beginning that I switch between